README.md

```diff
@@ -8,14 +8,67 @@ This action allows caching dependencies and build outputs to improve workflow ex
 
 See ["Caching dependencies to speed up workflows"](https://help.github.com/github/automating-your-workflow-with-github-actions/caching-dependencies-to-speed-up-workflows).
 
+## What's New in V2
+
+* Added support for caching multiple paths,
+
+```yaml
```

Comment:
For the sake of keeping this "What's New" section short, let's combine these into one example for multiple paths